src/context/defaultValues.tsx

```diff
@@ -28,8 +31,8 @@ const defaultUserValue = {
     number: "",
     floor: "",
   },
-  dispatch: (args: unknown): void => {
-    args = args;
+  dispatch: ({ type, payload }: UserAction): void => {
+    console.log(type, payload);
```

Comment:
console.log(type, payload);

Reply:
I added this line to keep eslint happy. If I keep the function empty or I don't use the function parameters, eslint will thrown warnings about that. However, this function will be overridden later so this code should not be executed at any time.